pkg/transfers/inbound/cleanup.go

```diff
@@ -25,13 +28,69 @@ func Cleanup(logger log.Logger, agent upload.Agent, dl *downloadedFiles) error {
 	if err := deleteFilesOnRemote(logger, agent, dl.dir, agent.ReturnPath()); err != nil {
 		el.Add(err)
 	}
+	if el.Empty() {
+		return nil
+	}
+	return el
+}
+
+// DeleteZeroByteFiles deletes empty ACH files if file older than value in config
+func DeleteZeroByteFiles(logger log.Logger, agent upload.Agent, dl *downloadedFiles, after time.Duration) error {
+	var el base.ErrorList
+	var fileInfosToCheck []os.FileInfo
+	inboundInfos, err := getFileInfos(dl.dir, agent.InboundPath())
+	if err != nil {
+		el.Add(err)
+	}
+	for i := range inboundInfos {
+		fileInfosToCheck = append(fileInfosToCheck, inboundInfos[i])
+	}
+
+	returnPathInfos, err := getFileInfos(dl.dir, agent.ReturnPath())
+	if err != nil {
+		el.Add(err)
+	}
+	for i := range returnPathInfos {
+		fileInfosToCheck = append(fileInfosToCheck, returnPathInfos[i])
+	}
+
+	for i := range fileInfosToCheck {
+		fileInfo := fileInfosToCheck[i]
+		path := filepath.Join(dl.dir, filepath.Base(fileInfo.Name()))
+		if shouldDeleteEmptyFile(fileInfo, after) {
+			err := deleteFile(agent, path)
```

Comment:
Why can't we just call `err := agent.Delete(path)`?

Reply:
Totally. I'll kill `deleteFile`